network-api/networkapi/mozfest/models.py

```diff
@@ -294,5 +264,68 @@ class MozfestLandingPage(MozfestPrimaryPage):
     def get_template(self, request):
         return "mozfest/mozfest_landing_page.html"
 
-    # Don't offer to use a wide tamplet
+    # Don't offer to use a wide tamplate
     settings_panels = Page.settings_panels
+
+    # Hero/Banner fields
```

Comment:
Yea if this was shared even more I'd probably created a related model but there's so much inheritance and shared components already I just want to keep this straight forward because I predict we will be breaking up the inheritance of the page models in this sprint at some point

Reply:
Sorry, yea. I didn't meant to suggest you should change anything. 

Just dreaming 😅